WalletWasabi.Gui/Assets/LegalDocuments.txt

```diff
@@ -82,7 +82,7 @@ You agree to indemnify, defend and hold us, our employees, agents, consultants,
 ===========================================
 
-Our trademarks, service marks, designs, logos, URLs, and trade names that are displayed on our Services we refer to in these Terms collectively as the “Materials”. We hereby grant you a limited, non-exclusive, revocable, royalty-free, non-transferable and non-sublicensable licence to access and use the Materials for your Services use. Such licence is subject to these Terms and does not permit any resale or exploitation of the Materials; the distribution, public performance or public display of any Materials; modifying or otherwise making any derivative uses of the Materials, or any portion thereof; or any use of the Materials other than for their intended purposes. The licence granted under this Section will automatically terminate and be revoked if we suspend or terminate your access to and use of the Services. You agree that the Service Provider will own exclusive rights, including all intellectual property rights, to any feedback, suggestions, ideas or other information or materials relating to the Service Provider or our Services that you provide, whether by email or otherwise, which we refer to in these Terms as “Feedback”. Any Feedback you submit is non-confidential and will become the sole property of the Service Provider. We will be entitled to the unrestricted use and dissemination of such Feedback for any purpose, commercial or otherwise, without acknowledgment or compensation to you. We reserve all rights not expressly granted herein.
+Our trademarks, service marks, designs, logos, URLs, and trade names that are displayed on our Services are referred to in these Terms collectively as the “Materials”. We hereby grant you a limited, non-exclusive, revocable, royalty-free, non-transferable and non-sublicensable licence to access and use the Materials for your Services use. Such licence is subject to these Terms and does not permit any resale or exploitation of the Materials; the distribution, public performance or public display of any Materials; modifying or otherwise making any derivative uses of the Materials, or any portion thereof; or any use of the Materials other than for their intended purposes. The licence granted under this Section will automatically terminate and be revoked if we suspend or terminate your access to and use of the Services. You agree that the Service Provider will own exclusive rights, including all intellectual property rights, to any feedback, suggestions, ideas or other information or materials relating to the Service Provider or our Services that you provide, whether by email or otherwise, which we refer to in these Terms as “Feedback”. Any Feedback you submit is non-confidential and will become the sole property of the Service Provider. We will be entitled to the unrestricted use and dissemination of such Feedback for any purpose, commercial or otherwise, without acknowledgment or compensation to you. We reserve all rights not expressly granted herein.
```

Comment:
GitHub shows that something is changed here, but I cannot tell what exactly...

Reply:
Before:
Our trademarks, service marks, designs, logos, URLs, and trade names that are displayed on our Services **we refer** to in these Terms collectively as the “Materials”
After:
Our trademarks, service marks, designs, logos, URLs, and trade names that are displayed on our Services **are referred** to in these Terms collectively as the “Materials”